docs/src/main/asciidoc/developer-guide/inc/_internal_design.adoc

```diff
@@ -7,7 +7,7 @@ AtlasMap consists of following 3 parts:
 2. <<internal-design-time-service,Design Time Service>>
 3. <<internal-runtime-engine,Runtime Engine>>
 
-<<internal-ui,Data Mapper UI>> is an Angular based web browser application. <<internal-design-time-service,Design Time Service>> is a set of REST API which provide background services to be used by the Data Mapper UI behind the scene. Data Mapper UI eventually produces data mapping definition file in XML or JSON format. Then <<internal-runtime-engine,Runtime Engine>> consumes that mapping definition as well as actual data payload and perform mapping.
+<<internal-ui,Data Mapper UI>> is a React based web browser application. <<internal-design-time-service,Design Time Service>> is a set of REST API which provide background services to be used by the Data Mapper UI behind the scene. Data Mapper UI eventually produces data mapping definition file in ADM archive format. Then <<internal-runtime-engine,Runtime Engine>> consumes that mapping definition as well as actual data payload and perform mapping.
```

Comment:
Would it be a good idea to change references to 'Data Mapper UI' or 'AtlasMap Data Mapper UI' to just 'AtlasMap UI'?

Reply:
It could be, but not in this PR